scene/gui/split_container.cpp

```diff
@@ -98,11 +98,10 @@ void SplitContainer::_resort() {
 	int sep = get_constant("separation");
 	Ref<Texture> g = get_icon("grabber");
 
-	if (dragger_visibility == DRAGGER_HIDDEN_COLLAPSED) {
+	if (dragger_visibility == DRAGGER_HIDDEN_COLLAPSED)
```

Comment:
Not sure about this change. IMO, the braces make it more readable, but that's probably just me.

Reply:
The entire Godot codebase seems very inconsistent in that regard. I have seem a lot of both brace and brace-less single line "if"s and "else"s.

Reply:
There are countless examples of this or that, this really shouldn't be an issue.

Reply:
I prefer braces. It makes harder to make an error. In some cases might be acceptable, but it makes no sense to pointlessly remove then. It also makes harder to see the diff, it shows something changed but you didn't actually changed them.

Reply:
I personally too. I'm currently testing out how far [I can push not using them](https://github.com/RayKoopa/godot/blob/33277ce16d9f4a85abf386400fed4b5c2847e89a/editor/editor_help_search.cpp#L342).

Reply:
Guess I'm alone here then, putting back the removed braces.